x/claim/keeper/claim.go

```diff
@@ -64,6 +65,18 @@ func (k Keeper) GetClaimRecords(ctx sdk.Context, chain types.Chain) ([]types.Cla
 	return claimRecords, nil
 }
 
+func (k Keeper) GetAllClaimRecords(ctx sdk.Context) ([]types.ClaimRecord, error) {
+	claimRecords := []types.ClaimRecord{}
```

Comment:
```suggestion
	var claimRecords []types.ClaimRecord
```
My IDE tells me to use this instead

Reply:
any specific reason? are they not functionally identical?

Reply:
your way is totally legit as well , I guess it is just preference , I am using `Goland` it always recommend me to do it this way, I will leave it to you to decide.